sample/todo-sync/src/com/cloudant/todo/TasksModel.java

```diff
@@ -70,10 +69,9 @@ public TasksModel(Context context) {
                 DATASTORE_MANGER_DIR,
                 Context.MODE_PRIVATE
         );
-        DatastoreManager manager = DatastoreManager.getInstance(path.getAbsolutePath());
         try {
-            this.mDatastore = manager.openDatastore(TASKS_DATASTORE_NAME);
-        } catch (DatastoreNotCreatedException dnce) {
+            this.mDocumentStore = DocumentStore.getInstance(new File(path, TASKS_DATASTORE_NAME));
+        } catch (DocumentStoreNotOpenedException dnce) {
```

Comment:
I wondered if we also needed to catch the `DocumentStoreNotCreatedException` here, but I see we only have a NotOpened, I thought we were having both types?
If we do we could probably just catch the `DocumentStoreException`?